tests/README.md

```diff
@@ -1,15 +1,81 @@
-# Starter tests
```

Comment:
This documentation for testing is not needed. It also contains some mistakes and unoptimal solutions.

I have prepared an extensive documentation for E2E testing and it will be present in Websight CMS docs after merging this PR:
https://github.com/websight-io/docs/pull/232

In README.md we can just a link to the official Websight CMS docs. Maintaining the same stuff in two places will be a burden.

Reply:
Can you please read it and explain what is unoptimal? This document is focused on the developer exprience, we want to be able to run e2e tests without Docker. In your proposal docker compose is needed. So you build project, build docker image, run docker compose and finally run tests.

In the case above, developer needs to buuld project and run tests :)

Reply:
After discussion, I agree that it's nice to have a README.md which dives into more technical details and presents a faster dev workflow for skilled devs.